### PR TITLE
Erm 1350 Added tests for AgreementLineSection FormInfo component 

### PR DESCRIPTION
**Pull request by MonireRasouli**

### CODE REVIEW

File: src/components/AgreementLineSections/FormInfo/FormInfo.test.js [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+      test('renders the Description field by id', () => {
+        const { getByTestId } = renderComponent;
+        expect(getByTestId('line-description')).toBeInTheDocument();
+      });
+
+      test('renders the Description field by id', () => {
+        const { getByTestId } = renderComponent;
+        expect(getByTestId('line-note')).toBeInTheDocument();
+      });
```
Comment (aditya-matukumalli) [inline src/components/AgreementLineSections/FormInfo/FormInfo.test.js:70]: Both the tests have the same decription.

Also, could you change this to use the getByRole textBox as you have used in the other tests? That way you can remove the data-testid passed to those Fields as well

File: src/components/AgreementLineSections/FormInfo/testResources.js [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+    'isLoading': false,
+    'lineId': '045e0540-48b3-461c-9e5a-06513ef40587',
+    'onSubmit': 'ƒ () {}' };
+export {
+    data,
+    handlers,
+    initialValues,
+    values
+};
+
```
Comment (aditya-matukumalli) [inline src/components/AgreementLineSections/FormInfo/testResources.js:145]: please indent this file.

File: src/components/AgreementLineSections/FormInfo/testResources.js [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+        'startDate': '2021-08-04',
+        'endDate': '2021-08-28',
+        'contentUpdated': null,
+        'haveAccess': true
+      }
+    };
+
+    const handlers = {
+      'isSuppressFromDiscoveryEnabled': 'ƒ () {}',
+      'onClose': 'ƒ () {}'
```
Comment (aditya-matukumalli) [inline src/components/AgreementLineSections/FormInfo/testResources.js:73]:
```suggestion
      isSuppressFromDiscoveryEnabled: () => {},
      onClose: () => {}
```

**Review by aditya-matukumalli — CHANGES_REQUESTED** [review]

File: src/components/AgreementLineSections/FormInfo/FormInfo.test.js (hunk 1 above)
Comment (MonireRasouli) [reply src/components/AgreementLineSections/FormInfo/FormInfo.test.js:70]: I have already used getByRole textbox for 'description' and 'note' in lines 25-33.

File: src/components/AgreementLineSections/FormInfo/testResources.js [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+    'contentUpdated': null,
+    'haveAccess': true
+  };
+
+  const values = { 'isEholdingsEnabled': true,
+    'isLoading': false,
+    'lineId': '045e0540-48b3-461c-9e5a-06513ef40587',
+    'onSubmit': 'ƒ () {}' };
+
+export {
```
Comment (aditya-matukumalli) [inline src/components/AgreementLineSections/FormInfo/testResources.js:141]: Missing indentation here

File: src/components/AgreementLineSections/FormInfo/testResources.js [hunk 5]
```diff
@@ -0,0 +1,147 @@
...
+  'startDate': '2021-08-04',
+  'endDate': '2021-08-28',
+  'contentUpdated': null,
+  'haveAccess': true
+};
+
+const values = { 'isEholdingsEnabled': true,
+  'isLoading': false,
+  'lineId': '045e0540-48b3-461c-9e5a-06513ef40587',
+  'onSubmit': 'ƒ () {}' };
```
Comment (aditya-matukumalli) [inline src/components/AgreementLineSections/FormInfo/testResources.js:139]: Please change this to a function 
```suggestion
  'onSubmit': () => {} };
```

**Review by aditya-matukumalli — CHANGES_REQUESTED** [review]
Please update the title of the PR and also indent the testResources file.

**Review by aditya-matukumalli — APPROVED** [review]

**Pull request merged**